### PR TITLE
Added some basic Workspace History

### PR DESCRIPTION
**Pull request by ross-p-smith**
# PR for issue #1811 

## What is being addressed

Displayed the Resource History properties in a list

Behold the splendor:
image

### CODE REVIEW

File: ui/app/src/components/shared/ResourceHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  function userFriendlyKey(key: String){
+    let friendlyKey = key.replaceAll('_', ' ');
+    return friendlyKey.charAt(0).toUpperCase() + friendlyKey.slice(1).toLowerCase();
+  }
+
+  return (
+    <div className={classNames.itemCell} data-is-focusable={true}>
+      <div className={classNames.itemContent}>
+        <div className={classNames.itemName}>{item.properties.display_name}</div>
+        <div className={classNames.itemIndex}>{`Item ${index}`}</div>
```
Comment (damoodamoo) [inline ui/app/src/components/shared/ResourceHistory.tsx:73]: think we can do without this?

File: ui/app/src/components/shared/ResourcePropertyPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        }
+    }
+
+    return(
+        <>
+            <Stack wrap horizontal>
+                <Stack.Item grow styles={stackItemStyles}>
+                    {props.header}
+                </Stack.Item>
+                <Stack.Item grow styles={stackItemStyles}>
```
Comment (damoodamoo) [inline ui/app/src/components/shared/ResourcePropertyPanel.tsx:30]:
```suggestion
                <Stack.Item grow={3} styles={stackItemStyles}>
```

**Comment by ross-p-smith** [comment]
> * The history list - shall we reverse the order, so the latest change is on top? Think that makes more sense as a 'version history' ?

@damoodamoo - can you give me a hand sorting by resourceVersion?

**Review by damoodamoo — APPROVED** [review]
LGTM!

**Pull request merged**